src/ralph/admin/fields.py

```diff
@@ -0,0 +1,154 @@
+# -*- coding: utf-8 -*-
+import re
+
+from django import forms
+from django.forms import ValidationError
+from django.forms.util import ErrorList
+from django.utils.safestring import mark_safe
+from django.utils.translation import ugettext as _
+
+
+class MultilineField(forms.CharField):
+    """
+    This field is a textarea which treats its content as many values seperated
+    by `separators`
+    Validation:
+        - separated values cannot duplicate each other,
+    """
+    widget = forms.Textarea
+    separators = ",|\n"
+
+    def __init__(self, allow_duplicates=True, *args, **kwargs):
+        self.allow_duplicates = allow_duplicates
+        super().__init__(*args, **kwargs)
+
+    def validate(self, values):
+        if not values and self.required:
+            error_msg = _(
+                "Field can't be empty. Please put the item OR items separated "
```

Comment:
"Field cannot be empty. Please insert an item (multiple items can be newline or comma separated)."